dags/solutions_team/solutionsteam_tf_se_nightly_supported.py

```diff
@@ -167,29 +150,9 @@
       runtime_version=RuntimeVersion.TPU_VM_TF_NIGHTLY_POD.value,
   )
 
-  embedding_dim = 32
-  tf_dlrm_v5p_8 = tf_config.get_tf_dlrm_config(
```

Comment:
is this intended to be deleted?

Reply:
yes, v5p will have sparsecore and dlrm models will be on pjrt and still work in progress to enable it, so it shouldn't have been added to the se tests, will create a bug to add it to the pjrt one once the sprasecore GA work is done